README.md

```diff
@@ -220,6 +220,7 @@ the value of the `master` and assumes it will be present during a real
 run. That means, in presence of an invalid `master`, `--check` may
 signal success but the actual play run fails.
 
+The `team` type uses `roundrobin` as the `runner` configuration.
```

Comment:
@thom311 is this an NM or teamd default? Manpages for both do not seem to specify a default and the role does not set the runner at all.

Reply:
the role doesn't support setting an options for the team. Hence, it creates a connection profile with "team.config" empty. That means, NM will call teamd with `-c ''` ... and teamd, will indeed then use "roundrobin": https://github.com/jpirko/libteam/blob/c7237377dead39ae4a711297203bacf7edb9fa41/teamd/teamd.c#L956

Reply:
I think the addition to the readme is correct. However, we should also add some options, otherwise, it it doesn't seem very useful to me.

Reply:
Good point. @a-mere-peasant please add a sentence like "No further configuration is supported at the moment.". Also the the branch needs to be rebased as described in https://git-scm.com/book/en/v2/Git-Branching-Rebasing

Reply:
>Good point. @a-mere-peasant please add a sentence like "No further configuration is supported at the moment.".

Ok,I'll add this .
Should there be an indication about 
https://github.com/tyll/network/tree/team_options
for people who want to use it ? or something referencing #97 ?

Reply:
Good question. In my opinion, there is no need for the references since the README is meant for users and not developers.